aiconfig-docs/docs/editor.md

```diff
@@ -0,0 +1,166 @@
+---
+sidebar_position: 3
+---
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+import constants from '@site/core/tabConstants';
+
+# AIConfig Editor
+
+**AIConfig Editor** is a locally hosted playground where you can visually create and edit prompts stored as [AIConfig JSON files](./basics). The editor is `model-agnostic`, `multimodal`, and `extensible` by design - it can support any generative AI model with text, image, and audio modalities. You can quickly transition from prototype to production using the AIConfig generated from AIConfig Editor. The AIConfig SDK enables you to execute the prompts and model parameters from the AIConfig in your application code.
+
+This guide covers the core features of AIConfig Editor and demonstrates how to:
+
+- [Set up the Editor](#set-up)
+- [Open the AIConfig Editor](#open)
+- [Edit and save AIConfigs](#edit-and-save)
+- [Run prompts](#run-prompts)
+- [Chain prompts](#chain-prompts)
+- [Create prompt templates](#prompt-templates)
+
+Want to get started quickly? Check out our [Getting Started Tutorial](./getting-started).
+
+---
+
+## Set Up {#set-up}
+
+1. Install the [AIConfig python package](https://pypi.org/project/python-aiconfig/) to use the AIConfig editor.
+
+<Tabs groupId="package-manager" queryString defaultValue={constants.defaultPythonPackageManager} values={constants.pythonPackageManagers}>
+<TabItem value="pip">
+
+```bash
+$ pip3 install python-aiconfig
+```
+
+</TabItem>
+
+<TabItem value="poetry">
+
+```bash
+$ poetry add python-aiconfig
+```
+
+</TabItem>
+
+</Tabs>
+
+:::note
+You need to install the python AIConfig package to create and edit your configs using the AIConfig Editor. You can still use the AIConfig Node SDK to interact with your config in your application code.
+:::
+
+2. Setup your API Keys required by the model providers.
+
+You will need to specify your API keys for the model providers (i.e. OpenAI, Google, HuggingFace) you plan to use. We recommend adding your API keys as environment variables so that they are accessible for all projects. The python library will automatically detect and use them without you having to write any code.
```

Comment:
Let's have links to each place to get API keys:

- search "GOOGLE_API_KEY" here: https://github.com/lastmile-ai/aiconfig/blob/main/cookbooks/Gemini/Gemini.ipynb
- rip we don't have this in our hugging face cookbook..., but the env var we need to set is `HUGGING_FACE_API_TOKEN` and the link to get it from is https://huggingface.co/docs/hub/security-tokens
- search ANYSCALE_ENDPOINT_API_KEY in https://github.com/lastmile-ai/aiconfig/blob/main/cookbooks/Anyscale/getting_started.ipynb
- you already have OPENAI_API_KEY in this doc so you're good for that